resolwe_bio/tools/bigwig_chroms_to_ucsc.py

```diff
@@ -91,8 +92,11 @@ def create_new_header(infile, mappings, outfile):
                                          ends=[x[1] for x in ints],
                                          values=[x[2] for x in ints])
                 elif chrom not in mappings:
-                    print(warning("UCSC mapping for sequence {} is missing. "
-                                  "This sequence will not be included in the bigWig file.".format(chrom)))
+                    seq_num += 1
+
+        if seq_num > 0:
+            print(warning("UCSC chromosome/conting mapping for {} sequence(s) is missing. "
```

Comment:
I guess it would be still nice to know which sequences are the missing ones, one may be interested in a particular sequence and want to know weather is missing duo to (a) missing mapping vs. (b) nothing i going on there...

Reply:
This works as long as one or two contigs are missing. In ERCC containing genomes, we have 92 sequences that are missing, making the process warning section too long.

Reply:
Could we at least `echo` them in stdout so we can get missing sequences without running the tool? In this case it would simply mean using another `print(sequences)`, i guess... Or do you see no value in it?

Reply:
I will add another `print` as you suggested, thank you!